invenio_communities/assets/semantic-ui/js/invenio_communities/community/featured-communities/FeaturedCommunities.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from "react";
+import { withCancel } from "react-invenio-forms";
+import { http } from "../../api/config";
+import { Grid } from "semantic-ui-react";
+import FeaturedCommunity from "./FeaturedCommunity";
+import PropTypes from "prop-types";
+
+export default class FeaturedCommunities extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            isLoading: false,
+            data: { hits: [] },
```

Comment:
```suggestion
            data: undefined,
```

Reply:
If data set to undefined it fails silently, no error trace in the console, I can have a look a bit later.

Reply:
what do you mean that if fails silently?

Reply:
It was failing because it was not returning anything when there were no hits, it's fixed now